### DISCUSSION TITLE
Add `media_channel_number` property in media_player

### DISCUSSION THREAD

**Discussion by Drafteed** [home-assistant/architecture#802] [Entity Models]
## Proposal

In addition to the `media_channel` property, which contains the name of the channel, I suggest adding `media_channel_number` or `media_channel_num` which can contains the integer number of the current channel. 

In the current implementation, there is nowhere to store the channel number, except for the `media_content_id` property, which can contain random data, such as the URI string of the current channel.

Some integrations may try to get the current channel number of media player (for example, voice assistants) and in the case of using `media_content_id` this can be difficult.

We can also display the channel number in the frontend to improve the UI experience.

## Integrations that could potentially use the channel number

- Bravia TV
- Channels
- Google Assistant
- LG Netcast
- LG webOS Smart TV
- Philips TV
- Samsung Smart TV
- Mediaroom
- Yandex Smart Home (custom integration)

**Comment by Drafteed** [comment]
❗️ Core PR: https://github.com/home-assistant/core/pull/83893

**Comment by nick2525** [comment]
Still valid, any news?
